bittensor/_axon/__init__.py

```diff
@@ -280,19 +274,19 @@ class AuthInterceptor(grpc.ServerInterceptor):
     def __init__(
         self,
         receiver_hotkey: str,
-        blacklist: Callable = None,
+        metagraph: "bittensor.metagraph",
```

Comment:
Is there a way to turn off the blacklist feature? Or will the expectation be that it will be on by default?